gobblin-metrics-libs/gobblin-metrics-base/src/main/avro/GaaSObservabilityEventExperimental.avsc

```diff
@@ -188,6 +188,38 @@
           }
         }
       ]
-    }]
+    },
+    {
+      "name": "datasetsWritten",
+      "type": [
+        "null",
+        {
+          "type": "array",
+          "items": {
+            "type": "record",
+            "name": "DatasetMetric",
+            "doc": "DatasetMetric contains bytes and records written by Gobblin writers for the dataset URN.",
+            "fields": [
+              {
+                "name": "datasetUrn",
+                "type": "string",
+                "doc": "URN of the dataset"
+              },
+              {
+                "name": "bytesWritten",
+                "type": "long",
+                "doc": "Number of bytes written for the dataset"
+              },
+              {
+                "name": "recordsWritten",
+                "type": "long",
+                "doc": "Number of records written for the dataset"
```

Comment:
when we say "records", would that be files in the case of `CopySource` / distcp?  if so, let's describe accordingly in the doc string... and possibly even in the name (e.g. `entitiesWritten`)

Reply:
Yup it's files written, will update to entitiesWritten

Reply:
Could also be rows for the mysql/espresso case